src/CodeGenerator.php

```diff
@@ -305,4 +318,39 @@ private static function generateServices(
             }
         }
     }
+
+    private static function generateEnumConstants(Map $filesByPackage, ProtoCatalog $catalog, int $licenseYear)
```

Comment:
Could you rename $filesByPackage to $inputFilesByPackage or something similar?

Reply:
Good idea. Done!